content/concepts/consensus-network/invariant-checking.md

```diff
@@ -0,0 +1,152 @@
+# Invariant Checking
+
+This article provides a high-level overview of invariant checking, why it exist, how it works, and lists active invariants.
+
+Like many safety features, we all hope that invariant checking never actually needs to do anything. However, it can be useful to understand the XRP Ledger's invariants because they define hard limits on the XRP Ledger's transaction processing, and to recognize the problem in the unlikely event that a transaction fails because it violated an invariant check.
+
+## Introduction
+
+Invariant checking is a safety feature of the XRP Ledger. It consists of a set of checks, separate from normal transaction processing, that guarantee that certain _invariants_ hold true across all transactions.
+
+Invariants should not trigger, but they ensure the XRP Ledger's integrity from bugs yet to be discovered or even created.
+
+| Term | Description |
+|-----------|-------------|
+| Invariant | A rule that should always, without exception, be true. For example, "New XRP cannot be created". |
+| Invariant Checking | In the XRP Ledger, a system where code automatically confirms that transaction processing does not break the invariants. If a transaction's execution would break an invariant, the invariant checking system fails the transaction. |
```

Comment:
I meant these definitions to be for us as writers, not for the reader. This doesn't have to be in the article. (If we had a glossary, that would be a different story, but we don't.)